Assets/Scripts/UI/Navigator/AltMarker.cs

```diff
@@ -0,0 +1,117 @@
+
+using PlayerController;
+using Unity.XR.CoreUtils;
+using UnityEngine;
+
+public class AltMarker : MonoBehaviour
+{
+    public GameObject marker;
+
+    [SerializeField] private GameObject _innerCircle;
+    [SerializeField] private GameObject _outerCircle;
+    [SerializeField] private Shader shader;
+    
+    private PlayerProfileService _playerProfileService;
+
+    // distance when fading should start
+    private float _fadeTriggerDistance = 6.0f;
+    // distance when marker should be invisible
+    private float _invisDistance = 3.0f;
```

Comment:
```suggestion
    private const  float InvisDistance = 3.0f;
```
could be declared as constant, as this value is never changed